lib/central_mail/utilities.rb

```diff
@@ -29,14 +29,17 @@ def log_submission(uploaded_object, metadata)
       pdf_total = number_pages.count
 
       title = uploaded_object.class.to_s.split('::').first
-      Rails.logger.info("#{title}: Submission success",
-                        'uuid' => metadata['uuid'],
-                        'source' => metadata['source'],
-                        'docType' => metadata['docType'],
-                        'consumer_id' => uploaded_object.consumer_id,
-                        'consumer_username' => uploaded_object.consumer_name,
-                        'pageCount' => page_total,
-                        'pdfCount' => pdf_total)
+      log_details = {
+        'uuid' => metadata['uuid'],
+        'source' => metadata['source'],
+        'docType' => metadata['docType'],
+        'consumer_id' => uploaded_object.consumer_id,
+        'consumer_username' => uploaded_object.consumer_name,
+        'pageCount' => page_total,
+        'pdfCount' => pdf_total
+      }.merge(metadata)
```

Comment:
As we discussed elsewhere, let's make sure to slice the metadata we want to keep, since there's potentially PII we shouldn't be logging by a full merge.

Reply:
I ultimately went with `slice`, but considered `except` for a minute. Slice seems safer, but we'll have to make a note somewhere if we add to our metadata in the future and expect it in our logs.